src/app/global/utils.nim

```diff
@@ -69,7 +69,7 @@ QtObject:
 
   proc hex2Dec*(self: Utils, value: string): string {.slot.} =
     # somehow this value crashes the app
-    if value == "0x0":
+    if value.find(re("0x0+$")) >= 0:
```

Comment:
I see that `re` handle a regex but I didn't find where is it defined. Can you please share some more info?